internal/pkg/cli/task_run.go

```diff
@@ -26,37 +36,57 @@ var (
 	taskRunEnvPromptHelp = fmt.Sprintf("Task will be deployed to the selected environment. "+
 		"Select %s to run the task in your default VPC instead of any existing environment.", color.Emphasize(config.EnvNameNone))
 	taskRunGroupNamePromptHelp = "The group name of the task. Tasks with the same group name share the same set of resources, including CloudFormation stack, CloudWatch log group, task definition and ECR repository."
+
+	fmtTaskFamilyName = "copilot-%s"
+	fmtRepoName       = "copilot-%s"
+	fmtImageURL       = "%s:%s"
+)
+
+const (
+	defaultImageTag       = "copilot"
```

Comment:
Can we re-use: https://github.com/aws/copilot-cli/blob/68929f73961e121c14e5caf77f4e0675e256c9b0/internal/pkg/cli/git.go#L13-L21

Reply:
do you think it makes sense if I first try to `getVersionTag`, if it fails (e.g. it is not a git repo), then I use this `defaultImageTag`?

Reply:
I think what we do with `svc deploy` is first checking if we can getVersionTag, if that fails we prompt for a tag value